mx_gate.py

```diff
@@ -253,7 +254,7 @@ def gate(args):
                 if mx.command_function('pylint')(['--primary']) != 0:
                     _warn_or_abort('Pylint not configured correctly. Cannot execute Pylint task.', args.strict_mode)
 
-        gate_clean(cleanArgs, tasks, tags=[Tags.build, Tags.style])
+        gate_clean(cleanArgs, tasks, tags=[Tags.build, Tags.fullbuild])
```

Comment:
I thought about that, too, but I think for now it's simpler to just do this "manually", i.e. every `build` step is also tagged `fullbuild`.

This feature would be more useful in e.g. the Graal gate, where we would like the `test` tag to implicitly select `build`. Anyway, if we really want that, we can do it in a separate change.